sidebars.js

```diff
@@ -168,10 +168,8 @@ module.exports = {
             //   items: [
             //     'php-sync-vs-async-start',
             //     'php-workers',
-            //
-            //     // todo: workflow interface
-            //     // todo: creating workflows
-            //
+            //     'php-workflow-interface',
```

Comment:
Uncomment and add them to sidebar per changes made here: https://github.com/temporalio/documentation/pull/251